frontend/src/components/pages/listings/index/Listings.tsx

```diff
@@ -4,6 +4,8 @@ import { Listing } from "@interfaces/listings";
 import { useQuery } from "@apollo/client";
 import { Grid, Typography, makeStyles, CircularProgress, Button } from "@material-ui/core";
 import Link from "next/link";
+import Image from "next/image";
+import _EmptyStreet from "public/illustrations/EmptyStreet.svg";
```

Comment:
Only need the underscore for stuff that doesn't start with a letter
```suggestion
import EmptyStreet from "public/illustrations/EmptyStreet.svg";
```